docs/docs/networks/picasso/governance.md

```diff
@@ -1,28 +1,73 @@
-# Governance
+# Picasso OpenGov
```

Comment:
not just picasso

Reply:
do you mean Composable aswell? that has another page